lib/chef/knife/bootstrap/templates/chef-full.erb

```diff
@@ -177,7 +177,7 @@ do_download() {
   else
     echo "-----> Installing Chef Omnibus (<%= @config[:channel] %>/<%= version_to_install %>)"
     do_download ${install_sh} $tmp_dir/install.sh
-    sh $tmp_dir/install.sh -P chef -c <%= @config[:channel] %> -v <%= version_to_install %>
```

Comment:
The P is actually for product so using project here doesn't really make sense

Reply:
I'm trying to use a combination of the following two options:

-`--bootstrap-project cinc`
-`--bootstrap-url" https://omnitruck.cinc.sh/install.sh "`

The knife plug-in used in WorkStation relies on Chef gem, which is a bit troubling if you want to turn the remote into a cinc.

Reply:
yeah that already has a meaning and it is the product.  don't want to rename that.  the ship already sailed.

Reply:
Does that mean that the option name should be product instead of project?

Reply:
yep

Reply:
thanks, i've changed to `bootstrap_product`